scripts/json-to-css.js

```diff
@@ -221,7 +221,7 @@ function createImport(files) {
 	files.forEach(function(file) {
 		code = code + '@import "' + file + '";\n';
 	});
-	// console.log(code);
+	console.log(code);
```

Comment:
Console logje :)

Reply:
Die hoort daar! :) dat is voor je console bui enkel builden van de package. Doet verder niks in je project.